apps/src/lib/config/ethereum.rs

```diff
@@ -6,25 +6,37 @@ use serde::{Deserialize, Serialize};
 /// Default [Ethereum JSON-RPC](https://ethereum.org/en/developers/docs/apis/json-rpc/) endpoint used by the oracle
 pub const DEFAULT_ORACLE_RPC_ENDPOINT: &str = "http://127.0.0.1:8545";
 
+/// The mode in which to run the Ethereum bridge.
+#[derive(Clone, Debug, Serialize, Deserialize)]
+pub enum Mode {
+    /// Run `geth` in a subprocess, exposing an Ethereum
+    /// JSON-RPC endpoint at [`DEFAULT_ORACLE_RPC_ENDPOINT`]. By default, the
+    /// oracle is configured to listen for events from the Ethereum bridge
+    /// smart contracts using this endpoint.
+    Managed,
+    /// Do not start a managed `geth` subprocess. Instead of the oracle
+    /// listening for events using a Ethereum JSON-RPC endpoint, an endpoint
+    /// will be exposed by the ledger itself for submission of Borsh-
+    /// serialized [`EthereumEvent`]s. Mostly useful for testing purposes.
+    EventsEndpoint,
+    /// Do not run any components of the Ethereum bridge
+    Off,
```

Comment:
NB: will add and test a mode for running the oracle but not the managed `geth` subprocess in a separate PR - https://github.com/anoma/namada/pull/253